changelog

```diff
@@ -1,3 +1,50 @@
+Version 0.10.12
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@reticent-rem, @TheGiraffe3, @tibetiroka, @TomGoodIdea, @warp-core)
+      * The job "Paradise Job: Wilderness Retreat" will no longer select a station as its destination. (@TheGiraffe3)
+      * The "Dangerous Games" missions will now only offer from planets within 100 hyperspace links of their destinations, instead of potentially in disconnected regions. (@warp-core)
+      * Fixed some log entry actions for Tim the stowaway that were malformed. (@warp-core)
+      * Added a missing passenger to the "Human to Hai Space - No Contact" mission. (@TheGiraffe3)
+      * Changed the description of the Tractor Beam to refer to "attractor" rather than "repulsor" technology. (@mOctave)
+      * The locations for distressed ships in Remnant rescue jobs one through three are now restricted to systems connected by hyperlane or wormhole to Arculus, so the objective ships can always make it back home. (@warp-core)
+    * Engine bugs:
+      * Fixed an issue where player escorts could start jumping around independently of the flagship. (@warp-core)
+      * Tooltip boxes will no longer be drawn for items without tooltip text in ship and outfit info displays. (@warp-core)
+      * When determining if there are sufficient resources to fire a weapon, only the values of resources firing the weapon actually consumes will be considered. (@warp-core)
+      * Exceptions thrown by std::copy in Files::Copy will now be caught, instead of crashing the game. (@TomGoodIdea)
```

Comment:
~~let's keep it human-readable maybe~~
```suggestion
      * Exceptions thrown when copying files will now be caught, instead of crashing the game. (@TomGoodIdea)
```